recipe/meta.yaml

```diff
@@ -39,7 +39,8 @@ source:
     - pycrypto_PR_200.diff
 
 build:
-  number: 10
+  skip: True    #[py>=38]
+  number: 11
   script:
     - set CL=/FI"%VCToolsInstallDir%INCLUDE\stdint.h" %CL%  # [win and py>=35]
     - export LDFLAGS="-headerpad_max_install_names"       # [osx]
```

Comment:
in the host are missing packages setuptools and wheel:
```
requirements:
  build:
   ....
  host:
    - python
    - pip
    - setuptools
    - wheel
  run:
...
```

Reply:
will update these.